src/commands/migrate.js

```diff
@@ -83,7 +83,7 @@ function migrationStatus(args) {
   }).catch(e => helpers.view.error(e));
 }
 
-function migrateSchemaTimestampAdd(args) {
+function migrateSchemaTimestampAdd (args) {
```

Comment:
@rtablada you want to remove these unnecessary whitespace changes?

Reply:
The changes seem to be from the eslint config in the project that was missing. I can try to turn off autoformat/fix in my config.